src/browser/new-browser.ts

```diff
@@ -226,13 +225,26 @@ export class NewBrowser extends Browser {
         config: BrowserConfig,
         capabilities: WebdriverIO.Capabilities,
     ): Promise<WebdriverIO.Capabilities> {
-        const browserNameLowerCase = config.desiredCapabilities?.browserName?.toLowerCase() as string;
+        const { getNormalizedBrowserName, installBrowser } = await import("../browser-installer");
+        const normalizedBrowserName = getNormalizedBrowserName(this._config.desiredCapabilities?.browserName);
+
+        if (!normalizedBrowserName) {
+            throw new Error(
+                [
+                    `Running auto local "${this._config.desiredCapabilities?.browserName}" is unsupported`,
+                    `Supported browsers: "chrome", "firefox", "safari", "edge"`,
+                ].join("\n"),
+            );
+        }
+
         const executablePath = await installBrowser(
-            this._config.desiredCapabilities?.browserName as SupportedBrowser,
-            this._config.desiredCapabilities?.browserVersion as string,
+            normalizedBrowserName,
```

Comment:
"installBrowser" now only wants normalized browser name (so we could pull out "invalid browser name" handling in order to throw relevant errors)